Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -685,7 +685,18 @@ bool GameList::AddShortcutToDesktop()
   if (FAILED(SHGetKnownFolderPath(FOLDERID_Desktop, KF_FLAG_NO_ALIAS, nullptr, &desktop)))
     return false;
 
-  const auto& game_name = game->GetLongName();
+  std::string game_name = game->GetName(Core::TitleDatabase());
+  // Sanitize the string by removing all characters that cannot be used in NTFS file names
+  game_name.erase(std::remove_if(game_name.begin(), game_name.end(),
+                                 [](char ch) {
+                                   const char illegal_characters[] = {'<',  '>', ':', '\"', '/',
```

Comment:
Perhaps make this `static constexpr char`, so this array doesn't get constructed on the stack for every execution of the lambda (which is currently what MSVC does)